android/docker/mlperf_mobile/Dockerfile

```diff
@@ -67,7 +67,7 @@ RUN yes | sdkmanager \
     "extras;android;m2repository" \
     "extras;google;m2repository" \
     "extras;google;google_play_services" \
-    "build-tools;29.0.3"
+    "build-tools;30.0.3"
```

Comment:
The build tool on line 58 is 29.0.3. Is it ok to change it to 30.0.3 here without changing in line 58 as well?